### ISSUE TITLE
[Regression] Project system now warns when a user has an explict facade reference and you reference a portable library

### ISSUE THREAD

**Issue by davkean** [dotnet/roslyn#4192]
1. Create a new console application targeting 4.5
2. Unload console application add:

```
<ItemGroup>
    <Reference Include="System.Collections" />
</ItemGroup>
```

3. Reload project
4. Add a reference to System-Runtime-based portable library

@Expected: No warnings
@Actual:

Severity	Code	Description	Project	File	Line
Warning		The referenced component 'System.Collections' could not be found.	ConsoleApplication6		

This regressed in VS 2015, and now results in ~10 warnings in Roslyn.

**Comment by Pilchie** [comment]
This requires manually editing the project file to add facade references.
